src/Tribe/Admin/Notices.php

```diff
@@ -514,7 +520,7 @@ public function undismiss_for_all( $slug ) {
 	 */
 	public function register( $slug, $callback, $arguments = [], $active_callback = null ) {
 		// Prevent weird stuff here
-		$slug = sanitize_title_with_dashes( $slug );
+		$slug = sanitize_key( $slug );
```

Comment:
Is this going to be back-compat? Just asking because `sanitize_key` doesn't add dashes...or was that the issue to begin with?

Reply:
We used mostly to avoid bugs with how WordPress register assets.